PureDOOM.h

```diff
@@ -22600,6 +22589,23 @@ void M_FinishReadThis(int choice)
     M_SetupNextMenu(&MainDef);
 }
 
+//
+// When a secret is found (ColleagueRiley)
```

Comment:
git history will have your name for your lines, no need to put it here :)

Reply:
Okay, my concern was ensuring that my code follows the DOOM license. Not making it seem like that part of the code is part of the source.

Reply:
Ok, fair enough. its not that important. But maybe put where this change originated from. 
```
// [PureDOOM]  or [PD] .
```
You can leave your name at the end it doesnt matter.